.goreleaser.yml

```
@@ -1,18 +1,45 @@
+# Visit https://goreleaser.com for documentation on how to customize this
+# behavior.
+
 before:
   hooks:
     - go mod tidy
-    - go test -v ./...
 builds:
-  - goos: [linux, darwin, windows]
-    goarch: [amd64]
-    binary: 'terraform-provider-msgraph{{if eq (printf "%.8s" .Version) "snapshot"}}_v{{.Version}}{{end}}'
+- env:
+  - CGO_ENABLED=0
+  goos:
+    - freebsd
+    - openbsd
+    - solaris
+    - windows
+    - linux
+    - darwin
+  goarch:
+    - amd64
+    - '386'
+    - arm
+    - arm64
+  ignore:
+    - goos: darwin
+      goarch: '386'
+    - goos: openbsd
+      goarch: arm
+    - goos: openbsd
+      goarch: arm64
+    - goos: freebsd
+      goarch: arm64
+  binary: '{{ .ProjectName }}_v{{ .Version }}'
```

An extra `v` in binary names as `terraform-provider-msgraph_vv0.1.3`

```suggestion
  binary: '{{ .ProjectName }}_{{ .Version }}'
```

This would affect the local plugin discovery: https://www.terraform.io/docs/extend/how-terraform-works.html#plugin-locations